src/main/java/org/openhab/tools/analysis/checkstyle/AboutHtmlCheck.java

```diff
@@ -89,6 +105,24 @@ protected void processFiltered(File file, List<String> lines) throws CheckstyleE
         }
     }
 
+    private boolean checkBuildPropertiesFile(File file, String property, String value) throws CheckstyleException {
+        if (!isEmpty(file)) {
+            try {
+                IBuild buildPropertiesFile = parseBuildProperties(file);
+                IBuildEntry binIncludes = buildPropertiesFile.getEntry(property);
+
+                return binIncludes != null && binIncludes.contains(value);
+
+            } catch (CheckstyleException e) {
+                logger.error("Error occured while processing {} file", BUILD_PROPERTIES_FILE_NAME, e);
+            }
+        } else {
+            logger.warn("{} file is included, but is empty", BUILD_PROPERTIES_FILE_NAME);
```

Comment:
I will add it in the report produced by the BuildPropertiesCheck, otherwise we might end up with multiple checks reporting the same problem.
I will remove the message at all.